lessons/intro/click/index.md

```diff
@@ -131,22 +131,35 @@ Přepínač `--help` přidává click sám.
 
 ## Argumenty
 
-Kromě přepínačů podporuje click i *argumenty*.
-Přepínače musí uživatel na řádce pojmenovat; argumenty se zadávají pozičně.
-Používají se ve dvou případech: pro povinné informace a pro argumenty, kterých
-může být libovolný počet.
+Kromě přepínačů podporuje click i [*argumenty*](http://click.pocoo.org/6/arguments/).
+Přepínače musí uživatel na řádce pojmenovat; argumenty se zadávají beze jména,
+ale záleží u nich na pořadí.
+Používají se ve dvou případech: pro povinné parametry a pro parametry, kterých
+může být zadán libovolný počet.
 Na všechno ostatní radši použijte přepínače.
 
+Například příkaz `cd` potřebuje jeden argument: jméno adresáře,
+do kterého má přepnout.
+Jeho rozhraní by v Clicku vypadalo takto:
+
 ```python
 @click.command()
 @click.argument('directory')
 def cd(directory):
-    """Change the current directory"""
+    """Create the specified directory"""
```

Comment:
Bogus docstring?

Reply:
I really need to focus more. Thanks for the catch.